src/vs/workbench/contrib/files/browser/views/explorerViewer.ts

```diff
@@ -275,6 +278,60 @@ export class FilesRenderer implements ICompressibleTreeRenderer<ExplorerItem, Fu
 	private _onDidChangeActiveDescendant = new EventMultiplexer<void>();
 	readonly onDidChangeActiveDescendant = this._onDidChangeActiveDescendant.event;
 
+	private readonly hoverDelegate = new class implements IHoverDelegate {
+
+		readonly placement = 'element';
+
+		get delay() {
+			return this.configurationService.getValue<number>('workbench.hover.delay');
+		}
+
+		constructor(
+			private readonly configurationService: IConfigurationService,
+			private readonly hoverService: IHoverService
+		) { }
+
+		showHover(options: IHoverDelegateOptions, focus?: boolean): IHoverWidget | undefined {
+			let element: HTMLElement;
+			if (options.target instanceof HTMLElement) {
+				element = options.target;
+			} else {
+				element = options.target.targetElements[0];
+			}
+
+			const child = element.children[0] as Element | undefined;
+			const childOfChild = child?.children[0] as HTMLElement | undefined;
```

Comment:
This looks very fragile, querying children via `querySelector('.some-class')` would be more solid

Reply:
Agreed, fixed.